src/apiV4.ts

```diff
@@ -2931,6 +2929,13 @@ export interface components {
       title: string;
       /** Number of donors who donated to the fundraising event */
       donor_count: number;
+      /** Company name if it is a sponsored fundraising event */
+      sponsoring_name: string;
+      links: {
+        rel: "sponsoring_logo";
+        href: string;
+        templated?: boolean;
+      }[];
```

Comment:
I don't understand what this has to do with the TypeScript update...

Reply:
I copy pasted the last state of apiV4.d.ts from .org to be in sync.